src/components/YourCompany/YourCompany.vue

```diff
@@ -199,10 +350,10 @@ export default class YourCompany extends Mixins(DateMixin, EntityFilterMixin, Le
 
   @Watch('getApprovedName')
   private onApprovedName ():void {
-    if (this.getApprovedName) {
+    if (this.getApprovedName && !this.isNumberedCompany) {
       this.correctNameChoices = [
         CorrectionTypes.CORRECT_NEW_NR,
-        CorrectionTypes.CORRECT_NAME,
+        this.isCorrection() && CorrectionTypes.CORRECT_NAME, // Only allow editable name changes for Corrections
```

Comment:
I think this will leave a falsey (undefined I think) element in the array.
It might be better to dynamically create the array (eg, with push/pop/slice/etc) instead.

Reply:
Ok, good point. Will check it out!

Reply:
Fixed in last commit